.github/workflows/test.yml

```diff
@@ -40,7 +59,6 @@ jobs:
         run: manifestoo -d . check-licenses
       - name: Check development status
         run: manifestoo -d . check-dev-status --default-dev-status=Beta
-        continue-on-error: true
```

Comment:
And probably this too until all dev status issues are fixed.

Reply:
I forced module dev statuses here. If you think ignoring option is better...